TwitterKit/TwitterKitTests/SocialTests/Syndication/Views/TWTRTimestampLabelTests.m

```diff
@@ -37,6 +39,9 @@ @implementation TWTRTimestampLabelTests
 - (void)setUp
 {
     [super setUp];
+    
+    [TWTRDateFormatters resetCache];
+    [TWTRDateFormatters setLocale:[NSLocale localeWithLocaleIdentifier:@"en_US"]];
```

Comment:
Hi @ra1028 ,

Use `[NSLocale currentLocale]` instead of using en_US locale so not to confuse with region dates.